src/lib/get-komodo-rewards/index.js

```diff
@@ -40,7 +41,14 @@ const getKomodoRewards = utxo => {
 	rewardPeriod -= 59;
 
 	// Calculate rewards
-	const rewards = Math.floor(satoshis / DEVISOR) * rewardPeriod;
+	let rewards = Math.floor(satoshis / DEVISOR) * rewardPeriod;
+
+	// Vote-KIP0001 resulted in a reduction of the AUR from 5% to 0.01%
+	// https://github.com/KomodoPlatform/kips/blob/main/kip-0001.mediawiki
+	// https://github.com/KomodoPlatform/komodo/pull/584
+	if (height >= N_S7_HARDFORK_HEIGHT) {
+		rewards = rewards / 5;
```

Comment:
Not exactly my expertise, but I guess it should be `rewards = rewards / 500;`, please correct me if I'm wrong.
Ref: https://github.com/KomodoPlatform/atomicDEX-API/blob/730aad8568f32a6aa1233f24f7f36e21e417f1fa/mm2src/coins/utxo.rs#L1640

Reply:
Thanks for the catch. That's correct. I think there should also be a `Math.floor()`

Like `rewards = Math.floor(rewards / 500)`